packages/sui-ssr/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@s-ui/ssr",
-  "version": "5.21.0",
+  "version": "5.21.0-beta.0",
```

Comment:
👀

Reply:
this will be change for the 5.22 in the release